src/reanimated2/animation/decay/utils.ts

```diff
@@ -61,8 +61,8 @@ export function isValidRubberBandConfig(
 ): config is RubberBandDecayConfig {
   'worklet';
   return (
+    !!config.rubberBandEffect &&
```

Comment:
I don't think double negation is necessary now.

Reply:
I'ts not, but our linter think it is
![image](https://github.com/software-mansion/react-native-reanimated/assets/56199675/4cddcc23-ba02-4539-a96b-9b9932dd57fd)

Reply:
:swiniok: